core/src/main/java/org/elasticsearch/index/engine/InternalEngine.java

```diff
@@ -1126,4 +1126,12 @@ public void onSettingsChanged() {
     public MergeStats getMergeStats() {
         return mergeScheduler.stats();
     }
+
+    @Override
+    public final void addIndices(Directory... directories) throws IOException {
```

Comment:
nit: since we call indexWriter.addIndexes, maybe use the same name here? alternatively we can de addDirectories, which might be a better name from an ES context (as Index means something else)